### PR TITLE
Bootstrapping recipe for inline-html

### PR DESCRIPTION
**Pull request by holtgrewe**

### CODE REVIEW

**Comment by holtgrewe** [comment]
To whom it may concern, this is ready for review now.

File: recipes/inner-html/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    - inline_html
+  commands:
+    - inline-html --help
+
+about:
+  home: https://github.com/zopyx/inline-html
+  license: GPL-2.0
+  license_family: GPL
+  # NB: Pending on upstream https://github.com/zopyx/inline-html/issues/2
+  # license_file: LICENSE.txt
```
Comment (bgruening) [inline recipes/inner-html/meta.yaml]: If this gets not merged, we need to include the license in the feedstock, GPL requires a copy of the license.

Comment (holtgrewe) [reply recipes/inner-html/meta.yaml]: No problem. Is this best done with a patch? Or is there dedicated functionality in conda for adding files?

**Comment by bgruening** [comment]
You add them to the recipe folder :)

**Comment by holtgrewe** [comment]
@bgruening OK, looks I managed doing this ;)

**Comment by holtgrewe** [comment]
@bgruening Do you know whom to ping to get this reviewed and merged?

**Comment by bgruening** [comment]
Sorry!

**Pull request merged**